app/src/main/java/com/example/concordia_campus_guide/Helper/IndoorPathHeuristic.java

```diff
@@ -16,6 +16,12 @@ public IndoorPathHeuristic(Context context){
         this.context = context;
     }
 
+    /**
+     * The heuristic is based on the distance between the goal and destination coordinate, the further a point from a destination, the bigger the heuristic it will have.
```

Comment:
The heuristic is based on the distance between the goal and destination coordinate, the further a point from a destination, the bigger the heuristic it will have.
```suggestion
     * The heuristic is based on the distance between the goal and destination coordinates, the further a point from a destination, the bigger the heuristic it will have.
```